motion-1/Overview.bs

```diff
@@ -92,68 +92,140 @@ In addition to the property-specific values listed in their definitions, all pro
 
 ISSUE: Add more details and examples.
 
-<h3 id="motion-path-property">Motion path: The 'motion-path' property</h3>
+<h3 id="offset-path-property">Define a path: The 'offset-path' property</h3>
 
 <pre class='propdef'>
-Name: motion-path
-Value: <<url>> | [ <<basic-shape>> | <<path()>> ] || <<geometry-box>> | none
-Initial: none
+Name: offset-path
+Value: <<url>> | <<angle>> && contain? | [ <<basic-shape>> | <<path()>> ] || <<geometry-box>> | none
+Initial: 0deg for <<angle>>, otherwise none
 Applies to: All elements. In SVG, it applies to <a>container elements</a> excluding the <{defs}> element and all <a>graphics elements</a>
 Inherited: no
 Percentages: n/a
 Computed value: as specified
 Media: visual
-Animatable: no
+Animatable: as specified for <a href="https://www.w3.org/TR/css3-transitions/#animatable-types">angle</a>, otherwise no
 </pre>
 
-Specifies the <a>motion path</a> the element gets positioned at. The exact element's position on the motion path is determined by the 'motion-offset' property. A <dfn>motion path</dfn> is either a specified path with one or multiple sub-paths or the geometry of a not styled basic shape. Each shape or path must define an <dfn>initial position</dfn> for the computed value of "0" for 'motion-offset' and an <dfn>initial direction</dfn> which specifies the rotation of the object on the initial position.
+Specifies the <a>path</a> the element gets positioned at. The element's exact position on the path is determined by the 'offset-distance' property. A <dfn>path</dfn> is either a specified path with one or multiple sub-paths or the geometry of a not styled basic shape. Each shape or path must define an <dfn>initial position</dfn> for the computed value of "0" for 'offset-distance' and an <dfn>initial direction</dfn> which specifies the rotation of the object on the initial position.
 
-In this specification a direction (or rotation) of 0 degree is equivalent to the direction of the positive x-axis in the object's <a>local coordinate system</a>. E.g a rotation of 0 degree points to the right side of the UA if the object and its ancestors have no transformation applied.
+In this specification, a direction (or rotation) of 0 degree is equivalent to the direction of the positive y-axis in the object's <a>local coordinate system</a>. E.g a rotation of 0 degree points to the upper side of the UA if the object and its ancestors have no transformation applied.
 
 Values have the following meanings:
 
-<dl dfn-for="motion-path">
-  <dt><<basic-shape>></dt>
+<dl dfn-for="offset-path">
+  <dt><<angle>></dt>
   <dd>
-    The motion path is a basic shape as specified in CSS Shapes [[!CSS-SHAPES]]. The initial position and the initial direction for basic shapes are defined as follows:
+    The path is a closed line segment that starts from the current position of the element and has the degree of the angle as specified in <<angle>> from the Y-axis. The path ends at the edge of the containing block.
```

Comment:
I think 'closed' is generally used with paths and means that the path starts and ends at the same point. Given that paths can be used here I'd avoid that word.

Ray is incorrect too - as you point out that would imply that it extends infinitely in one direction.

I think the best terminology would just be 'line segment'. Segments are finite by definition.